Code/Framework/AzToolsFramework/AzToolsFramework/API/PythonLoader.cpp

```diff
@@ -40,9 +41,12 @@ namespace AzToolsFramework::EmbeddedPython
         AZ::IO::PathView libPythonName = AZ::IO::PathView(PYTHON_SHARED_LIBRARY_PATH).Filename();
         AZ::IO::FixedMaxPath pythonVenvLibPath = pythonVenvPath / "lib" / libPythonName;
 
-        m_embeddedLibPythonModuleHandle = AZ::DynamicModuleHandle::Create(pythonVenvLibPath.StringAsPosix().c_str(), false);
-        bool loadResult = m_embeddedLibPythonModuleHandle->Load(false, true);
-        AZ_Error("PythonLoader", loadResult, "Failed to load %s.\n", libPythonName.StringAsPosix().c_str());
+        if (AZ::IO::SystemFile::Exists(pythonVenvLibPath.StringAsPosix().c_str()))
+        {
+            m_embeddedLibPythonModuleHandle = AZ::DynamicModuleHandle::Create(pythonVenvLibPath.StringAsPosix().c_str(), false);
+            bool loadResult = m_embeddedLibPythonModuleHandle->Load(false, true);
+            AZ_Error("PythonLoader", loadResult, "Failed to load %s.\n", libPythonName.StringAsPosix().c_str());
```

Comment:
The `libPythonName` is just a filename without path separators (such as `libPython3.10.so`), so the `StringAsPosix()` call only makes an unnecessary copy of the string.
```suggestion
            AZ_Error("PythonLoader", loadResult, "Failed to load %s.\n", libPythonName.c_str());
```